collect_app/src/main/java/org/odk/collect/android/provider/FormsProvider.java

```diff
@@ -214,7 +215,7 @@ public synchronized Uri insert(@NonNull Uri uri, ContentValues initialValues) {
             if (!values.containsKey(FormsColumns.FORM_MEDIA_PATH)) {
                 String pathNoExtension = filePath.substring(0,
                         filePath.lastIndexOf('.'));
-                String mediaPath = pathNoExtension + "-media";
+                String mediaPath = pathNoExtension + MEDIA_SUFFIX;
```

Comment:
Extracting a constant is a step in the right direction, but I think we're still half-way through. 

I suggest that we have a private constant, and that third parties use methods in FileUtils that return media paths. That way, we hide the implementation details of "building media folder paths", and we reduce code duplication, which makes changing it easier in the future.